sagemaker-clarify/time_series_deepar/time_series_deepar.ipynb

```diff
@@ -0,0 +1,2293 @@
+    "\n",
+    "5. [Analysis Config](#Analysis-Config)\n",
+    "\n",
+    "    a. [Retrieve Config From s3](#Retrieve-Config-From-s3)\n",
+    "        \n",
+    "    b. [Display Config](#Display-Config)\n",
+    "\n",
+    "6. [Explainability Results](#Explainability-Results)\n",
+    "\n",
+    "    a. [Retrieve Results From s3](#Retrieve-Results-From-s3)\n",
+    "    \n",
+    "    b. [Display Results](#Display-Results)\n",
+    "\n",
+    "7. [Clean Up](#Clean-Up)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1985cfd8-7425-4008-bd30-0d11220f186f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Summary\n",
+    "\n",
+    "This notebook is created to demonstrate the invocation of a SageMaker Clarify explainability job for a time series DeepAR forecasting model. Given a real model and a training dataset, a processingJob will be create to analyze the SHAP score for each feature attribute."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75aeddef-bdba-4e57-bccb-e08172703bcc",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Prerequisites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e87fc8d-d827-495f-8a32-4c4b93ab1ba7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Install Mercury\n",
+    "\n",
+    "If not already installed, the following cell will install the `mercury` package in order to display the `analysis_config.json` and explainability job output within the notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c32654dc-cd02-4be8-9e7b-e38d16bb44dc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mWARNING: Ignoring invalid distribution -otocore (/local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33mWARNING: dj-rest-auth 3.0.0 does not provide the extra 'with-social'\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33mWARNING: Ignoring invalid distribution -otocore (/local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
+      "awscli 1.31.7 requires botocore==1.33.7, but you have botocore 1.29.165 which is incompatible.\n",
+      "awscli 1.31.7 requires s3transfer<0.9.0,>=0.8.0, but you have s3transfer 0.6.2 which is incompatible.\n",
+      "sagemaker 2.214.3 requires boto3<2.0,>=1.33.3, but you have boto3 1.26.83 which is incompatible.\u001b[0m\u001b[31m\n",
+      "\u001b[0m"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install mercury -q"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af541390-8a5c-4a5f-9cf0-d7a2b27d11b6",
+   "metadata": {},
+   "source": [
+    "## Install Development SDK\n",
```

Comment:
Notebook doesn't use development SDK anymore, please update this section.

Reply:
Updated.